src/components/TodoList.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Todo from './Todo';
+
+import './TodoList.css';
+
+function TodoList({ todoList }) {
+  return (
+    <ul className="todo__list">
+      {todoList.map(todo => (
+        <li className="todo__item" key={todo.id}>
+          <Todo {...todo} />
+        </li>
```

Comment:
```suggestion
          <Todo {...todo} key={todo.id} />
```
li можно перенести внутрь компонента Todo

Reply:
Исправил